lua/entities/gmod_wire_clutch.lua

```diff
@@ -277,6 +277,7 @@ function ENT:ApplyDupeInfo(ply, ent, info, GetEntByID)
 		   Ent1 ~= Ent2 and
 		   hook.Run( "CanTool", ply, WireLib.dummytrace(Ent1), "ballsocket_adv" ) and
 		   hook.Run( "CanTool", ply, WireLib.dummytrace(Ent2), "ballsocket_adv" ) then
+
```

Comment:
Newline

Reply:
Strange. I did not add it. Not in its local files.

Reply:
I fixed indent.

Reply:
I blundered. My mistake.